File: MyFinance/ClientApp/src/app/models/monthPayment.ts
import { Calc } from "./calc.model";

export class MonthPayment {
  constructor(
    public id: string = '00000000-0000-0000-0000-000000000000',
    public number: number = 0,
    public payment: number = 0,
    public persentegesPyment: number = 0,
    public persentege: number = 0,
    public debitPayment: number = 0,
    public debit: number = 0,
    public credit: Calc = new Calc(),
  ) { }
}

File: MyFinance.Data/MonthPayment.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace MyFinance.Data
{
    public class MonthPayment
    {
        public Guid Id { get; set; }
        public int Number { get; set; }
        public decimal Payment { get; set; }
        public decimal PersentegesPyment { get; set; }
        public decimal Persentege { get; set; }
        public virtual Credit Credit { get; set; }
    }
}
